pkgs/tools/text/xml/basex/default.nix

```diff
@@ -25,23 +25,15 @@ stdenv.mkDerivation rec {
   dontBuild = true;
 
   installPhase = ''
-    mkdir -p "$out"
-    cp -r * "$out"
-
     # Remove Windows batch files (unclutter $out/bin)
-    rm -f "$out"/bin/*.bat
+    rm ./bin/*.bat
 
-    # Move some top-level stuff to $out/share/basex (unclutter $out)
-    mkdir -p "$out/share/basex"
-    mv "$out"/*.txt "$out/share/basex/"
-    mv "$out"/webapp "$out/share/basex/"
+    mkdir -p $out/share/basex" "$out/share/applications"
```

Comment:
I cannot build the `basex` package on my machine because of the unmatched `"` in this line.

Reply:
The details of the failed builds here show the same problem: 
![image](https://user-images.githubusercontent.com/26806/97095457-84bee180-1657-11eb-9962-765e8345ca3b.png)